sdk/test-utils/perfstress/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:12
+
+WORKDIR /app
+
+COPY package*.json ./
+RUN npm ci --only=production
+
+COPY . .
+
+ENTRYPOINT ["node", "test"]
```

Comment:
I'm not currently using this file, but it will be relevant for: https://github.com/azure/azure-sdk-for-js/issues/8063